hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/launcher/TestContainersLauncher.java

```diff
@@ -238,7 +234,8 @@ public void testSignalContainerEvent()
     when(containerId.getApplicationAttemptId().getApplicationId())
         .thenReturn(appId);
 
-    Whitebox.setInternalState(spy, "running", dummyMap);
+    spy.running.clear();
+    spy.running.putAll(dummyMap);
```

Comment:
Same as the above. Also, we should replace `dummyMap.size()` as `spy.running.size()` in `assertEquals`.

Reply:
Thanks for your suggestion, I will modify the code.